e2e/scripts/st_pyplot.py

```diff
@@ -20,3 +20,17 @@
 data = np.random.normal(1, 1, size=100)
 plot = pyplot.plot(data)
 st.pyplot()
+pyplot.clf()
+
+st.set_option("deprecation.showPyplotGlobalUse", False)
+np.random.seed(0xDEADBEEF)
```

Comment:
Curious, does the random seed matter here? It doesn't look like the latter two tests use image snapshots.

Maybe the tests would be clearer if you just didn't bother regenerating random data, but plotting the same data 3 times? (not 100% on this)

Reply:
Probably not the biggest deal, but I removed the regeneration.